code/src/index.css

```diff
@@ -11,3 +11,26 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+#checkbox {
+  width: 1.3em;
+  height: 1.3em;
+  background-color: white;
+  border-radius: 50%;
+  vertical-align: middle;
+  border: 1px solid #ddd;
+  appearance: none;
+  -webkit-appearance: none;
+  outline: none;
+  cursor: pointer;
+}
+
+#checkbox:checked {
+  text-decoration: line-through;
+  background-color: #DF7861;
+}
+
+#checkbox:checked+label {
+  text-decoration: line-through;
```

Comment:
Very nice looking checkbox, was so surprised that it wasn't a radiobutton. Also nice touch with the line-through, it makes the list extra readable.